src/components/preview/components/page-preview-component.tsx

```diff
@@ -61,9 +61,9 @@ class PagePreviewComponent extends React.Component<Props, State> {
     const { classes } = this.props;
 
```

Comment:
```suggestion
```

Reply:
This is unused after deleting the class from the returned jsx element.

Reply:
done.